code/__defines/belly_modes_ch.dm

```diff
@@ -40,3 +40,7 @@
 #define AT_FLAG_ORES			0x40
 #define AT_FLAG_CLOTHES			0x80
 #define AT_FLAG_FOOD			0x100
+
+//Vorespawn flags
+#define VS_FLAG_ABSORB_YES		0x1
```

Comment:
Use the defined flags, don't hardcode numbers later on.

0 is not a valid flag as it prevents further usages